ecosystem/manager.py

```diff
@@ -80,13 +84,41 @@ def dispatch_check_workflow(
 
     def get_projects_by_tier(self, tier: str) -> None:
         """Return projects by tier.
-
         Args:
             tier: tier of ecosystem
         """
         repositories = ",".join([repo.url for repo in self.dao.get_repos_by_tier(tier)])
         set_actions_output([("repositories", repositories)])
 
+    def update_badges(self):
+        """Updates badges for projects."""
+        current_dir = os.path.dirname(os.path.abspath(__file__))
+        badges_folder_path = "{}/../badges".format(current_dir)
+
+        tests = []
+        community_projects = [
+            repo.name for repo in self.dao.get_repos_by_tier("COMMUNITY")
+        ]
+        tests_list = [
+            repo.tests_results for repo in self.dao.get_repos_by_tier("COMMUNITY")
+        ]
+
+        for i in tests_list:
+            test_result = [str(repo.passed) for repo in i]
+            if "False" not in test_result:
+                tests.append(True)
+            else:
+                tests.append(False)
+
+        for project, tests_out in zip(community_projects, tests):
+            tests_passed = tests_out
+            color = "blueviolet" if tests_passed else "gray"
+            url = f"https://img.shields.io/static/v1?label=Qiskit&message={project}&color={color}"
```

Comment:
also lets do `https://img.shields.io/static/v1?label=Qiskit&message=Ecosystem&color={color}`

Reply:
If we put Ecosystem, means we can't control someone to put badge on it's repo even if its project doesn't it inside our db